lib/ebay_request/base.rb

```diff
@@ -106,7 +106,7 @@ def prepare(url)
 
       if url.port == 443
         http.use_ssl = true
-        http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+        http.verify_mode = OpenSSL::SSL::VERIFY_PEER
```

Comment:
Actually, you can just remove this line altogether because it is default value per docs: http://ruby-doc.org/stdlib-2.4.1/libdoc/net/http/rdoc/Net/HTTP.html
> default value of #verify_mode is set as OpenSSL::SSL::VERIFY_PEER.

Reply:
done, thanks!